sample_cdr_georef/sample_cdr_georef/server.py

```diff
@@ -325,3 +421,4 @@ def register_system():
     if args.mode == 'process':
```

Comment:
@Sorrento110 think you could add the 'getter of results' calls to the 'process' path as well please?